### PR TITLE
chore:attempt 1

### PR DESCRIPTION
**Pull request by JakeBalakrishna**

### CODE REVIEW

File: src/Screens/MealOverview.tsx [hunk 1]
```diff
@@ -1,80 +1,105 @@
...
+    color: "#1B1D1E"
   },
-  percentageView: {
-    paddingTop: 250,
-    width: "100%"
+  textSubTitle: {
+    fontSize: 14,
+    fontWeight: "400",
+    color: "#333638",
+    paddingTop: 8,
```
Comment (mattwilson02) [inline src/Screens/MealOverview.tsx:57]: use gap inside your container

File: src/Screens/MealOverview.tsx [hunk 2]
```diff
@@ -1,80 +1,105 @@
...
   },
-  cardWrapper: {
-    flexDirection: "row",
-    marginHorizontal: 83,
-    justifyContent: "space-between"
+  statTitle: {
+    fontSize: 14,
+    fontWeight: "700",
+    paddingTop: 32,
+    paddingBottom: 23,
```
Comment (mattwilson02) [inline src/Screens/MealOverview.tsx:64]: i would just use 24 lol

File: src/Screens/MealOverview.tsx [hunk 3]
```diff
@@ -1,80 +1,105 @@
...
+    alignSelf: "flex-start"
   },
-  statText: {
-    fontSize: 16,
+  //same with this css as the comment below should be in its own component
+  mealPercentText: {
+    fontSize: 32,
     fontWeight: "700",
-    marginVertical: 25
+    textAlign: "center",
```
Comment (mattwilson02) [inline src/Screens/MealOverview.tsx:36]: add alignItems to the container and then you won't need this

File: src/Screens/MealOverview.tsx [hunk 4]
```diff
@@ -1,80 +1,105 @@
...
-    height: 40,
-    //ooft
-    paddingTop: 500,
-    paddingHorizontal: 30,
-    alignContent: "center",
-    position: "absolute",
-    zIndex: 1
+  mealsOverviewSubText: {
+    color: "#333638",
+    textAlign: "center",
```
Comment (mattwilson02) [inline src/Screens/MealOverview.tsx:41]: add alignItems to the container and then you won't need this

File: src/Screens/MealOverview.tsx [hunk 5]
```diff
@@ -1,80 +1,105 @@
...
+    paddingBottom: 16
   },
-  cardWrapper: {
-    flexDirection: "row",
-    marginHorizontal: 83,
-    justifyContent: "space-between"
+  statTitle: {
+    fontSize: 14,
+    fontWeight: "700",
+    paddingTop: 32,
```
Comment (mattwilson02) [inline src/Screens/MealOverview.tsx:63]: Bring your Statistics title inside the card view, then add this padding to the cardView instead

File: src/Screens/MealOverview.tsx [hunk 6]
```diff
@@ -85,42 +110,53 @@ export const MealOverview = () => {
...
-            </View>
+          {/* I've left this as inline because it needs to be changed 
+          its padding horizontal 93 because the view holding the two cards ignore the 
+          width restriction of the container for some reason idk why and i need help*/}
+          <View
+            style={{
+              flexDirection: "row",
+              gap: 8,
+              alignSelf: "center",
+              paddingHorizontal: 93
```
[the remark this answers is not in the document]
Comment (JakeBalakrishna) [reply src/Screens/MealOverview.tsx:148]: i did this and left a comment here in the code :)

**Review by mattwilson02 — APPROVED** [review]

**Pull request merged**